kodilla-stream/src/main/java/com/kodilla/stream/world/Country.java

```diff
@@ -0,0 +1,38 @@
+package com.kodilla.stream.world;
+
+import java.math.BigDecimal;
+
+public final class Country {
+    private final String nameCountry;
+    private final BigDecimal population;
+
+    public Country(String nameCountry, BigDecimal population) {
+        this.nameCountry = nameCountry;
+        this.population = population;
+    }
+
+    public BigDecimal getPeopleQuantity(){
+        return population;
+    }
+
+    public String getNameCountry() {
+        return nameCountry;
+    }
+
+    public BigDecimal getPopulation() {
+        return population;
+    }
```

Comment:
metoda getPeopleQuantity robi to samo co getPopulation. Jedna jest do wyrzucenia